pkg/ring/lifecycler.go

```diff
@@ -634,6 +858,18 @@ func (i *Lifecycler) changeState(ctx context.Context, state IngesterState) error
 	return i.updateConsul(ctx)
 }
 
+// getLastRing returns a copy of the last ring saved.
+func (i *Lifecycler) getLastRing() *Desc {
+	i.countersLock.Lock()
+	defer i.countersLock.Unlock()
+
+	if i.lastRing == nil {
+		return nil
+	}
+
+	return i.lastRing.Clone()
```

Comment:
Do we still need to have the Clone method? I believe we no longer modify the returned instance?